Tests/Controller/Annotations/AbstractParamTest.php

```diff
@@ -28,7 +28,6 @@ public function setUp()
     public function testInterface()
     {
         $this->assertInstanceOf('FOS\RestBundle\Controller\Annotations\ParamInterface', $this->param);
-        $this->assertInstanceOf('Symfony\Component\DependencyInjection\ContainerAware', $this->param);
```

Comment:
Imho, checking for a trait is checking for an implementation detail, should be avoided